### PR TITLE
Add country to the API with hacky fips='0'

### PR DESCRIPTION
**Pull request by TomGoBravo**
This PR addresses https://trello.com/c/dckJRBsA/574-add-usa-aggregation-to-api

Schema auto-generated files updated with `./run.py api update-schemas`

## Tested

This change has a unittest which passes. I also ran `python pyseir/cli.py build-all --generate-api-v2 --location-id-matches 'iso1:us$'` and browsed the files I found afterwards using `find output`.

### CODE REVIEW

File: libs/pipelines/api_v2_paths.py [hunk 1]
```diff
@@ -88,7 +92,12 @@ def bulk_flattened_timeseries_data(self, file_type, state: Optional[str] = None)
     def single_summary(self, region_summary: can_api_v2_definition.RegionSummary, file_type):
         if self.level is AggregationLevel.STATE:
             return self.region_subdir / f"{region_summary.state}.{file_type.suffix}"
-        if self.level in (AggregationLevel.COUNTY, AggregationLevel.CBSA, AggregationLevel.PLACE):
+        if self.level in (
+            AggregationLevel.COUNTY,
+            AggregationLevel.CBSA,
+            AggregationLevel.PLACE,
+            AggregationLevel.COUNTRY,
```
Comment (ghop02) [inline libs/pipelines/api_v2_paths.py:99]: I think we can use the country code for the key - I think /country/US.json is clearer and potentially more stable if we end up changing the key

**Review by ghop02 — CHANGES_REQUESTED** [review]
I'd like to change the key for the individual country but other than that looks good!

File: libs/pipelines/api_v2_paths.py (hunk 1 above)
Comment (TomGoBravo) [reply libs/pipelines/api_v2_paths.py:99]: Done in ccfc14cb

**Review by ghop02 — APPROVED** [review]
This looks good to me! 🚢

**Pull request merged**